tools/binlog/deploy.md

```diff
@@ -0,0 +1,513 @@
+
+It is recommended to deploy TiDB-Binlog using TiDB-Ansible. If you just want to do a simple testing, you can deploy TiDB-Binlog using a Binary package.
+
+## Deploy TiDB-Binlog using TiDB-Ansible 
+
+### Step 1: Download TiDB-Ansible
+
+1. Use the TiDB user account to log in to the central control machine and go to the `/home/tidb` directory. The information about the branch of TiDB-Ansible and the corresponding TiDB version is as follows. If you have questions regarding which version to use, email to [info@pingcap.com](mailto:info@pingcap.com) for more information or [file an issue](https://github.com/pingcap/tidb-ansible/issues/new).
+
+    | tidb-ansible branch | TiDB version | Note |
+    | ------------------- | ------------ | ---- |
+    | release-2.0 | 2.0 version | The latest 2.0 stable version. You can use it in the production environment. |
+    | release-2.1 | 2.1 version | The latest 2.1 stable version. You can use it in the production environment (recommended). |
+    | master | master version | This version includes the latest features with a daily update. |
+
+2. Use the following command to download the corresponding branch of TiDB-Ansible from the [TiDB-Ansible project](https://github.com/pingcap/tidb-ansible) on GitHub. The default folder name is `tidb-ansible`.
+
+    - Download the 2.0 version:
+        
+        ```bash
+        $ git clone -b release-2.0-new-binlog https://github.com/pingcap/tidb-ansible.git
+        ```
+
+    - Download the 2.1 version:
+
+        ```bash
+        $ git clone -b release-2.1 https://github.com/pingcap/tidb-ansible.git
+        ```
+
+    - Download the master version:
+
+        ```bash
+        $ git clone https://github.com/pingcap/tidb-ansible.git
+        ```
+        
+### Step 2: Deploy Pump
+1. Modify the `tidb-ansible/inventory.ini` file.
+
+    1. Set `enable_binlog = True` to start `binlog` of the TiDB cluster.
+
+        ```ini
+        ## binlog trigger
+        enable_binlog = True
+        ```
+
+    2. Add the deployment machine IPs for `pump_servers`.
+
+        ```ini
+        ## Binlog Part
+        [pump_servers]
+        172.16.10.72
+        172.16.10.73
+        172.16.10.74
+        ```
+
+        Pump retains the data of the latest 7 days by default. You can modify the value of the `gc` variable in the `tidb-ansible/conf/pump.yml` file and remove the related comments: 
+        
+        ```yaml
+        global:
+          # an integer value to control the expiry date of the binlog data, which indicates for how long (in days) the binlog data would be stored
+          # must be bigger than 0
+          # gc: 7
+        ```
+
+        Make sure the space of the deployment directory is sufficient for storing Binlog. For more details, see [Configure the deployment directory](../../dev/how-to/deploy/orchestrated/ansible.md#configure-the-deployment-directory). You can also set a separate deployment directory for Pump.
```

Comment:
Let's use the following link path rule because we are refactoring documentation these days. This rule can avoid possible issues.

```suggestion
        Make sure the space of the deployment directory is sufficient for storing Binlog. For more details, see [Configure the deployment directory](/dev/how-to/deploy/orchestrated/ansible.md#configure-the-deployment-directory). You can also set a separate deployment directory for Pump.
```